docs/tutorial/working-with-frames-and-series/index.md

```diff
@@ -68,15 +67,16 @@ The `patients.date_of_birth` column is a patient series; it has one row per pati
 
 ## Work with event data
 
-### Import the `medications` table
+Working with event data means working with many different tables.
```

Comment:
Not sure about this.  You can work with event data through working with a single table.  I read this as implying that you need to use many different tables to work with event data.

Reply:
Fair point. I've updated this to read:

> Unlike patient data, there are many sources of event data: the `medications` table, for example. To work with the `medications` table, first import it into the sandbox.